### DISCUSSION TITLE
Uploading larger size file issue

### DISCUSSION THREAD

**Discussion by liza-nt** [filamentphp/filament#8900] [Help]
I am uploading videos of more than 200 MB video using FileUpload to an s3 bucket and after changing max size and livewire config I am able to upload to larger file but what's happening now is now I can not access my video by bucket public URL but i can those video with bucket name URL... I can able to access it if it is a smaller size file through my s3 bucket public url

**Comment by leandrocfe** [comment]
any console errors?

**Reply by liza-nt** [reply]
Failed to load resource: the server responded with a status of 500 ()
but i can access that larger size(235mb) video through bucket name url but with public url it is saying access is denied
but I can access small size video(78mb) through both public and bucket name url

**Reply by Khant-Nyar** [reply]
![image](https://github.com/filamentphp/filament/assets/61083917/ed2dc222-da91-4089-a4af-525e93c3072a)

**Comment by votkapower** [comment]
Have you able to solve this?
